testing/Makefile

```diff
@@ -0,0 +1,34 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+IMG = gcr.io/minigo-testing/minigo-testing
+TAG := $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)-$(shell git diff | openssl sha -sha256 | cut -c -6)
+DIR := ${CURDIR}
+
+all: build
+
+build:
+	mkdir -p staging
+	cp ../requirements-cpu.txt staging/requirements-cpu.txt
```

Comment:
do you need to copy the python files in here?

Reply:
Nope. Those are pulled in via bootstrap.sh from github. I pull in the requirements so I can preload them (which isn't strictly required).